### ISSUE TITLE
incorrent image orientation 

### ISSUE THREAD

**Issue by themantalope** [alessiomaffeis/vue-picture-input#133]
Hi,

When displaying a preview image, the orientation is not correct. I'm not sure what needs to be done to fix it. The file on my computer is below:

![IMG_1731](https://user-images.githubusercontent.com/7599879/94351557-82994100-001f-11eb-912c-05714020264e.jpg)

As you can see, the orientation of the image should be such that the width < height, and the page is upright. Even when selecting this image on the [codepen](https://codesandbox.io/s/github/alessiomaffeis/vue-picture-input-example), the previewed image is displayed in the incorrect orientation (should be rotated 90 degrees) 

Any recommendations on how to fix this would be much appreciated.

**Comment by alessiomaffeis** [comment]
Hi @themantalope, I think I've found out why this happens: the image has an original landscape orientation and an EXIF flag set to rotate it 90°. When I built the component, I had to manually detect the EXIF orientation and correct it when drawing the image to the canvas, but that doesn't seem to be necessary on modern browsers anymore, since they now do it out of the box. 

What's happening here is that the component applies the rotation correction on top of the browser's one. I will push a fix for it!
